test/Interop/Cxx/operators/member-inline.swift

```diff
@@ -1,6 +1,9 @@
 // RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
 //
 // REQUIRES: executable_test
+//
+// We can't yet call member functions correctly on Windows.
+// XFAIL: OS=windows-msvc
```

Comment:
Ditto, could you add a link to the bug in the comment?

Reply:
Oops, I missed this. Done in https://github.com/apple/swift/pull/32699.